src/style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.input {
+  display: block;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300;
+  font-size: 16px;
+  font-family: inherit;
+}
+
+.input-small {
+  height: 42px;
+  margin-top: 20px;
+  padding-left: 33px;
+  font-family: inherit;
+  font-size: 14px;
+}
+
+.icon {
+  display: block;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin: 25 0 0 26;
```

Comment:
The margin property should have units for the values, such as 'px'. Update it to 'margin: 25px 0 0 26px;' 📏